### PR TITLE
Edit readme

### PR DESCRIPTION
**Pull request by k-doering-NOAA**
# What is the feature?

* Add R universe badge to readme
* Move developer install instructions to collaborative workflow (just link it here)

Addresses https://github.com/NOAA-FIMS/collaborative_workflow/issues/98

# How have you implemented the solution?

# Does the PR impact any other area of the project?

# How to test this change

Note the link should work on the readme after https://github.com/NOAA-FIMS/collaborative_workflow/pull/146 is merged in.

# Developer pre-PR checklist

- [x] I relied on GitHub actions to :test_tube: things for me while I sat on the :couch_and_lamp:.

### CODE REVIEW

File: README.md
Comment (kellijohnson-NOAA) [inline README.md:14]: I was thinking that this badge would go in line with the other badges.

**Review by kellijohnson-NOAA — APPROVED** [review]
Thanks @k-doering-NOAA for putting so much work into organizing this information. I love the move of the developer information to the collaborative workflow. This will help new users a lot I think so that they are not overwhelmed by the amount of information present. I made one comment about the location of the badge but I will leave that up to you.

**Comment by k-doering-NOAA** [comment]
Great suggestion @kellijohnson-NOAA ! I changed the badge location. Feel free to merge in when https://github.com/NOAA-FIMS/collaborative_workflow/tree/codespaces is also ready to merge in.

**Comment by kellijohnson-NOAA** [comment]
I squashed the commits and will merge into main once the checks are complete.

**Pull request merged**
